index.html

```diff
@@ -49,5 +49,167 @@ <h1 class="headline-title">Hey there. I'm Rachel</h1>
             <button type="button"><img src="./img/Union.png"></button>
         </div>
     </section>
-    </body>
-    </html>
\ No newline at end of file
+    <!--WORKS SECTION-->
+    <section>
+        <div class="work">
+            <h1>My Recent Works</h1>
+            <div class="indicator-image">
+            <img src="./img/Indicator.svg" alt="">
+            </div>
+            <article>
+                <div class="works-card">
+                    <div class="works-card-image">
+                        <img src="./img/Img Plaholder.svg" alt="">
+                    </div>
+                    <h2 class="works-card-title">Multi-Post Stories Gain+Glory</h2>
+                    <ul class="works-card-skills">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
```

Comment:
- [ ] kindly fix the background color & text-color for all of the `works-card-skills` to match figma design

Figma          | Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/192094886-6fbed92b-993e-4776-8323-890ce6d71f6f.png) | ![image](https://user-images.githubusercontent.com/89282221/192094804-f70b98cc-6632-4618-b278-f9da86339e17.png)